client/client_interfaces.go

```diff
@@ -196,10 +196,10 @@ type SystemAPIClient interface {
 // VolumeAPIClient defines API client methods for the volumes
 type VolumeAPIClient interface {
 	VolumeCreate(ctx context.Context, options volume.CreateOptions) (volume.Volume, error)
```

Comment:
Decided to split create into its own PR.

The options are still part of the API and removal was not trivial to me. Will follow-up.